doc/Language/regexes.pod6

```diff
@@ -472,16 +472,16 @@ character that is not whitespace.
 
 =head3 X<C<\d> and C<\D>|regex,\d;regex,\D>
 
-C<\d> matches a single digit (Unicode property C<N>) and C<\D> matches a
-single character that is not a digit.
+C<\d> matches a single decimal number (Unicode property C<Nd>) and C<\D> matches a
```

Comment:
I'd prefer the "digit" term to appear, if only as mnemonic. Full name in the unicode standard is comething like digit with decimal radix, but I'd very much prefer "digit" to "number" here.
Also, what it said initially is not really false. `N` includes `Nd`, so it might be worth the while to mention it.

Reply:
> I'd prefer the "digit" term to appear, if only as mnemonic. Full name in the unicode standard is something like digit with decimal radix, but I'd very much prefer "digit" to "number" here.

It looks like the Unicode name is "Number, decimal digit" (according to [tbl. 4-4](https://www.unicode.org/versions/Unicode14.0.0/ch04.pdf#G124206), anyway) and that's the name that we use on the [syntax](https://docs.raku.org/language/syntax#Ordinary_identifiers) page.  Since that still has the mnemonic "digit" in it, I changed to that here.

> Also, what it said initially is not really false. `N` includes `Nd`

Yeah, but it `N` is broader: `/<:N>/` matches things like `' ⑤'`, which aren't matched by `/\d/` (and are matched by `/\D/').